### PR TITLE
Updating dependencies, checking for breaking changes

### PR DESCRIPTION
**Pull request by waridrox**
Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] tests pass -- `rake test`
* [ ] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [ ] pull request are descriptively named
* [ ] if possible, multiple commits squashed if they're smaller changes
* [ ] reviewed/confirmed/tested by another contributor or maintainer

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
     "ace-builds": "jywarren/ace-builds#v1.2.4",
     "bootstrap-css": "jozefizso/bower-bootstrap-css#~2.3.2",
-    "d3": "mbostock-bower/d3-bower#~3.3.13",
-    "flot": "flot/flot#~0.7",
-    "fontawesome": "FortAwesome/Font-Awesome#~4.5.0",
-    "getusermedia.js": "addyosmani/getUserMedia.js#v1.0rc2",
-    "jasmine": "pivotal/jasmine#~2",
-    "jasmine-ajax": "jasmine/jasmine-ajax#v3.2.0",
+    "d3": "^3.5.4",
+    "flot": "~0.8.0-alpha",
```
Comment (jywarren) [inline package.json:12]: This looks great. The graphs work on all the prior interfaces? Great work!

**Review by jywarren — APPROVED** [review]
Just making sure flot works and then we can merge!!

File: package.json (hunk 1 above)
Comment (waridrox) [reply package.json:12]: Yes, I've checked locally as well as on gitpod, the live graph works as expected at the `/capture` route

**Comment by waridrox** [comment]
Should I squash the commits too ?

**Comment by jywarren** [comment]
No it's ok, GitHub squashes them upon merging, the way we have it set up. Merging!!!

**Comment by jywarren** [comment]
Great job!!!!

**Pull request merged**